docs/sources/tasks/collect-logs-in-kubernetes.md

```diff
@@ -0,0 +1,323 @@
+
+```alloy
+loki.write "default" {
+  endpoint {
+    url = "http://localhost:3100/loki/api/v1/push"
+  }
+
+  endpoint {
+    url = "https://logs-us-central1.grafana.net/loki/api/v1/push"
+
+    // Get basic authentication based on environment variables.
+    basic_auth {
+      username = env("<REMOTE_WRITE_USERNAME>")
+      password = env("<REMOTE_WRITE_PASSWORD>")
+    }
+  }
+}
+
+loki.source.file "example" {
+  // Collect logs from the default listen address.
+  targets = [
+    {__path__ = "/tmp/foo.txt", "color" = "pink"},
+    {__path__ = "/tmp/bar.txt", "color" = "blue"},
+    {__path__ = "/tmp/baz.txt", "color" = "grey"},
+  ]
+
+  forward_to = [loki.write.default.receiver]
+}
+```
+
+For more information on configuring logs delivery, refer to [loki.write][].
+
+## Collect logs from Kubernetes
+
+{{< param "PRODUCT_NAME" >}} can be configured to collect all kinds of logs from Kubernetes:
+
+1. System logs
+1. Pods logs
+1. Kubernetes Events
+
+Thanks to the component architecture, you can follow one or all of the next sections to get those logs. Once you have followed the [Configure logs delivery](#configure-logs-delivery) to ensure collected logs can be written somewhere, jump to the relevant sections.
+
+### System logs
+
+To get the system logs, you should use the following components:
+1. [local.file_match][]: Discovers files on the local filesystem
+1. [loki.source.file][]: Reads log entries from files
+1. [loki.write][]: Send logs to the Loki endpoint. You should have configured it in the [Configure logs delivery](#configure-logs-delivery) section
+
+Here is an example using those stages.
+
+```alloy
+// local.file_match discovers files on the local filesystem using glob patterns and the doublestar library. Expose an array of file path.
+local.file_match "node_logs" {
+  path_targets = [{
+      // Monitor syslog to scrape node-logs
+      __path__  = "/var/log/syslog",
+      job       = "node/syslog",
+      node_name = env("HOSTNAME"),
+      cluster   = <CLUSTER_NAME>
+  }]
+}
+
+// loki.source.file reads log entries from files and forwards them to other loki.* components.
+// You can specify multiple loki.source.file components by giving them different labels.
+loki.source.file "node_logs" {
+  targets    = local.file_match.node_logs.targets
+  forward_to = [loki.write.<WRITE_COMPONENT_NAME>.receiver]
+}
+```
+
+Replace the following values:
+
+- _`<CLUSTER_NAME>`_: The label for this specific Kubernetes cluster, such as `production` or `us-east-1`.
+- _`<WRITE_COMPONENT_NAME>`_: The name of your `loki.write` component, such as `default`.
+
+### Pods logs
+
+{{< admonition type="tip" >}}
+You can get pods logs through the log files on each node. In this guide, you will get the logs through the Kubernetes API because it doesn't require system privileges for {{< param "PRODUCT_NAME" >}}.
```

Comment:
Unfortunately, as the [docs](https://grafana.com/docs/alloy/latest/reference/components/loki.source.kubernetes/) for `loki.source.kubernetes` state:

> Because loki.source.kubernetes uses the Kubernetes API to tail logs, it uses more network traffic and CPU consumption of Kubelets than loki.source.file.

Do you think we should mention both `loki.source.kubernetes` and `loki.source.file` in the doc? I'm worried that folks with high traffic will attempt to use the wrong component.